Add note to One-to-many relationship (#495)
subquery/documentation: docs/build/graphql.md

}
```

::: warning Note

> `@derrivedFrom` is required and cannot be the `id` field, without this there is no constraint that `Account` is only associated to one person. This would then make it a many-to-many relationship, please follow the many-to-many steps below.

Reviewer: ```suggestion
`@derivedFrom` is required and cannot be the `id` field, without this there is no constraint that `Account` is only associated to one person. If you omit the `@derivedFrom`, then this relationship would instead be a many-to-many relationship as described below.
```
